<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-FNPC0HE" failures="1" tests="2" name="LoginPageTest" time="16.592" errors="0" timestamp="2025-02-12T22:01:52 IST">
  <testcase classname="org.qa.gittracker.tests.LoginPageTest" name="testLoginPageTitle" time="5.752"/>
  <testcase classname="org.qa.gittracker.tests.LoginPageTest" name="testLogin" time="6.408">
    <failure type="java.lang.AssertionError" message="expected [https://git-tracker-web-dev.herokuapp.com/dispatch/open] but found [https://git-tracker-web-dev.herokuapp.com/account/login]">
      <![CDATA[java.lang.AssertionError: expected [https://git-tracker-web-dev.herokuapp.com/dispatch/open] but found [https://git-tracker-web-dev.herokuapp.com/account/login]
at org.qa.gittracker.tests.LoginPageTest.testLogin(LoginPageTest.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
</testsuite> <!-- LoginPageTest -->
